signers/signer-cosmos/src/helpers.ts

```diff
@@ -133,7 +136,7 @@ export const executeCosmosTransaction = async (
   }
 };
 
-export function getsignedTx(
+export async function getsignedTx(
```

Comment:
`getsignedTx` now returns a promise so it should be used with `await` in `wallets/provider-walletconnect-2/src/signers/cosmos.ts`

Reply:
Fixed it